src/main/steps/common/form/fields/link.njk

```diff
@@ -1 +1 @@
- <a class="govuk-link" href="{{getContent(field.link)}}" target ="_blank">{{getContent(field.label)}}</a>
\ No newline at end of file
+ <a class="{{field.classes}}" href="{{field.href}}" >{{getContent(field.label)}}</a>
```

Comment:
any specific reason for removing target?

Reply:
target allows to open it in another tab. In future if such need is required we can create a target link